lib/Message.php

```diff
@@ -75,12 +75,11 @@ function getBodyAsString() {
             return '';
         }
         $contentLength = $this->getHeader('Content-Length');
-        if (null === $contentLength) {
-            return stream_get_contents($body);
-        } else {
+        if (is_int($contentLength) || ctype_digit($contentLength)) {
```

Comment:
It does. I didn't realize it was an extension. I'll add it to the `require` list to make sure this is captured